src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -493,7 +545,19 @@ def _start_worker_agent(self) -> None:  # pragma: no cover
 
     @property
     def worker_id(self) -> str:
-        cmd_result = self.send_command("cat /var/lib/deadline/worker.json  | jq -r '.worker_id'")
+        if self.configuration.operating_system.name == "AL2023":
+            cmd_result = self.send_command(
+                "cat /var/lib/deadline/worker.json  | jq -r '.worker_id'"
```

Comment:
nit: this can be simplified to just:

```suggestion
                "jq -r '.worker_id' /var/lib/deadline/worker.json"
```

Reply:
Updated and tested.